features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/AndroidTracer.kt

```diff
@@ -41,8 +57,11 @@ class AndroidTracer internal constructor(
     writer: Writer,
     random: Random,
     private val logsHandler: LogHandler,
-    private val bundleWithRum: Boolean
-) : DDTracer(config, writer, random) {
+    private val bundleWithRum: Boolean,
+    coreTracer: CoreTracer
+) : DDTracer(config, writer, random), Tracer {
+
+    private val otelTracer = OtelTracer("otel",coreTracer)
```

Comment:
are we going to support both OpenTracing and Otel in the same tracer?

Reply:
That's the idea @ganeshnj is following on iOS so we need to stay consistent here. Later we can deprecate all the methods from old tracer interface and then remove it.